src/main/java/mate/academy/spring/dto/UserResponseDto.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.spring.dto;
+
+public class UserResponseDto {
+    private long id;
```

Comment:
```suggestion
    private Long id;
```
default value of `long` is 0, for `Long` is null